node/service/src/lib.rs

```diff
@@ -552,9 +552,6 @@ where
 		Some(shared_authority_set.clone()),
 	);
 
-	let shared_epoch_changes = babe_link.epoch_changes().clone();
-	let slot_duration = babe_config.slot_duration();
```

Comment:
We still need this?

Reply:
I mean, you removed `slot_duration` and the code still needs it, so it will fail to compile or?

Reply:
`slot_duration` initialized at [L524](https://github.com/paritytech/polkadot/blob/20906a4f7778a3645815c891a39b92043d113795/node/service/src/lib.rs#L524)

Reply:
I thought you meant in general if we still needed this in `babe_config` 😅 Yes, `slot_duration` was duplicated in this function.

Reply:
Ahh, I should have checked all the code! ty @acatangiu